### PR TITLE
changes points 7 and 8

### PR DESCRIPTION
**Pull request by eugenividal**
Hi @Robinlovelace 

In this pull request, you will find the changes regarding points 7 and 8 in the paper (marked in blue). Point 9 did not require any change.

Please review the changes at point 7. All papers suggested (plus one additional by Winters) are now included in Table 1 and commented on in the text. The Cyclist Routing Algorithm for Network Connectivity (CRANC) tool is applied to the Boston region (according to the paper). However, I found this website rc.nau.edu/cranc applied to Arizona, which is actually hard to understand. Not sure if we should add in Table 1 that this tool includes online maps publicly available for Arizona. 

Any question please let me know here.

Eugeni

### CODE REVIEW

**Comment by Robinlovelace** [comment]
Taking a look now, thanks Eugeni!

File: paper.qmd [hunk 1]
```diff
@@ -180,7 +180,7 @@ In @sec-discussion, we discuss limitations and possible future improvements to t
 
 Tools for estimating geographical distribution of cycling potential have advanced substantially in recent years.
 Developed by researchers and developers in academic, public and private sectors, they have evolved from simple area-based static models tailored to specific regions to more complex and (in some cases) more generalizable tools.
-These tools can be considered as a specific application of the concept of planning support systems (PSS) , which are designed to help planners and decision-makers to make more informed decisions [@geertman2009].
+These tools can be considered as a specific application of the concept of planning support systems (PSS), which are designed to help planners and decision-makers to make more informed decisions [@geertman2009].
```
Comment (Robinlovelace) [inline paper.qmd:183]: :+1: good fix!

File: paper.qmd [hunk 2]
```diff
@@ -206,6 +206,7 @@ tools = tools |>
 
 ```{r}
 #| label: tbl-tools
+#| tbl-cap: "Summary of existing tools for estimating cycling potential."
```
Comment (Robinlovelace) [inline paper.qmd:209]: :+1:

File: paper.qmd [hunk 3]
```diff
@@ -225,17 +226,20 @@ Three years later, Transport for London created the Analysis of Cycling Potentia
...
 The tool does not use origin-destination data and cannot estimate cycling potential on specific routes [@transport_for_london_analysis_2010].
 
-Other tools that generate areal data to prioritize investment include a study that generated a 'prioritization index' for Montreal, Canada [@larsen_build_2013], a 'usage intensity index' based on stated preferences in Belo Horizonte, Brazil, and a Cycling Potential Tool (CPT) for Scotland [@phillips_development_2017].
+Other tools that generate areal data to prioritize investment include a study that generated a<span style="color: blue;"> 'bikeability index' for the Metro Vancouver region [@winters_mapping_2013] and later called Bike Score and available for over 160 US and Canadian cities [@winters_bike_2016] -- its methodology is [publicly available](https://www.walkscore.com/bike-score-methodology.shtml); a </span>'prioritization index' for Montreal, Canada [@larsen_build_2013]; a 'usage intensity index' based on stated preferences in Belo Horizonte, Brazil [@zhang_prioritizing_2014]; and a Cycling Potential Tool (CPT) for Scotland [@phillips_development_2017].
```
Comment (Robinlovelace) [inline paper.qmd:229]: Why the 'span' bit here @eugenividal, does it work in PDF version?

File: paper.qmd [hunk 4]
```diff
@@ -225,17 +226,20 @@ Three years later, Transport for London created the Analysis of Cycling Potentia
...
 The CPT is composed of the base environmental module — based on eight weighted factors (population density, hilliness, physical barriers, access to services, existing cycling mode share, distance to work and school, and road speed) — and the quality of service module, which evaluates each area of interest based on eight factors (surface condition, adjacent cyclists, comfort factor, conflict, distance between junctions, slope, access to services, and origin/destination).
 
-A more recent areal-based approach is the designed the Gross Potential for Cycling (GPC) tool to prioritize areas for cycling infrastructure and other cycling measures [@silva_gross_2021].
+A more recent areal-based approach is the designed Gross Potential for Cycling (GPC) tool to prioritize areas for cycling infrastructure and other cycling measures  [@silva_gross_2021].
```
Comment (Robinlovelace) [inline paper.qmd:232]: :+1: typo fix.

File: paper.qmd [hunk 5]
```diff
@@ -225,17 +226,20 @@ Three years later, Transport for London created the Analysis of Cycling Potentia
...
 The GPC uses two sets of indicators: population-based indicators (age, potential demand density, employment density and motorization rate) and area-based indicators (accessibility, education, public transport, connectivity, land use mix and relative performance).
 These indicators are calculated considering topography, road hierarchy, and average congestion and presented on a scale from 5 to 10.
 They are then combined into an overall score, weighted by their impact on cycling according to the literature.
 The ranking of areas by the GPC are [publicly available](https://boost.up.pt/en/ferramentas/gpc) and its practical value has been assessed through workshops [@silva_revealing_2022].
+<span style="color: blue;">The GPC tool has also been used to investigate the combination of factors contributing to high levels cycling potential [@lopes_impact_2021].
```
Comment (Robinlovelace) [inline paper.qmd:237]: Aha OK this is to highlight your changes, great!

**Review by Robinlovelace — APPROVED** [review]

File: paper.qmd (hunk 5 above)
Comment (eugenividal) [reply paper.qmd:237]: Exactly!

**Pull request merged**